### ISSUE TITLE
Epic: Implement the `hedera-admin-service` modules

### ISSUE THREAD

**Issue by rbair23** [hashgraph/hedera-services#4220]
Original
- [x] https://github.com/hashgraph/hedera-services/issues/4255
- [x] #4463 
- [x] #4464 
- [x] #4465 

MVP
- [x] #6195

Nice to haves
- [ ] #6234
- [x] update to respond to comments on PR #4464
- [ ] Implement security (dependent on #6131)
- [x] WritableFreezeStore for SwirldDualState access
